format_spec/filters/float_scale.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Float Scaling Filter
+---
+
+The float scale filter stores its input data (with floating point type) as integer values for more compressed storage.
+For example, given `data` as a NumPy float64 array, scale = 2.13, offset = 15.445, and a specified byte width of 4:
+  ```
+  data = np.random.rand(npts)
+  data_b = data.view(np.float64)
+  new_data_b = np.zeros(npts, dtype=np.int32) # byte width is 4
+  scale = 2.13
+  offset = 15.445
+  
+  for i in range(0, len(data)):
+    new_data_b[i] = (data_b[i] - offset) / scale
+  ```
+
+  
+On read, the float scaling filter will reverse the scale factor and offset, restoring the original floating point data, with a potential loss of precision. Given the previous example:
+   ```
+   restored_data = np.zeros(npts, dtype=np.float64)
+   for i in range(0, len(data)):
+     restored_data[i] = (new_data_b[i] * scale) + offset
+   ```
```

Comment:
You may want to add a more meaningful example here that highlights some of the pros/cons of this filter. If possible, I would also recommend not adding code snippets (since we the description to make sense regardless of which language people are programming in), and instead use just equations.

Small example with meaningful data:
scale = 0.25
offset = 10.0
byte_size = 2
input_data (double) = [10.0, 10.2500, 10.7540, 11.0001]
stored_data (16 bit integer) = [0, 1, 3, 4]
output_data (double) = [10.0, 10.2500, 10.7500, 11.0]